### PR TITLE
[6.4.z] Upgrade moving to Python 3 and makefile change

### PR DESCRIPTION
**Pull request by jyejare**
This Pr Contains :
1. Upgrade moving to python3 - > CherryPick of #254 
2. The makefile fix for maxlinelength to 99

### CODE REVIEW

File: upgrade/satellite.py [hunk 1]
```diff
@@ -168,12 +167,11 @@ def satellite6_upgrade():
         postup_time = datetime.now().replace(microsecond=0)
         logger.highlight('Time taken for Satellite Upgrade - {}'.format(
             str(postup_time-preup_time)))
-    set_hammer_config()
     # Rebooting the satellite for kernel update if any
     reboot(180)
     host_ssh_availability_check(env.get('satellite_host'))
     # Test the Upgrade is successful
-    hammer('ping')
+    run('hammer ping')
```
Comment (jyejare) [inline upgrade/satellite.py]: @ntkathole , Removed warn_only here.

File: upgrade/satellite.py [hunk 2]
```diff
@@ -270,6 +268,5 @@ def satellite6_zstream_upgrade():
     reboot(180)
     host_ssh_availability_check(env.get('satellite_host'))
     # Test the Upgrade is successful
-    set_hammer_config()
-    hammer('ping')
+    run('hammer ping', warn_only=True)
```
Comment (ntkathole) [inline upgrade/satellite.py]: @jyejare shall we remove it here as well?

Comment (jyejare) [reply upgrade/satellite.py]: Its ok, even if we keep this, It is not going to have any impact.

Comment (jyejare) [reply upgrade/satellite.py]: @ntkathole , Removed in all PRs.

**Pull request merged**